source/_components/broadlink.markdown

```diff
@@ -478,3 +478,51 @@ This is the code we need to transmit again to replicate the same remote function
 The "status" : "OK" at the end is a feedback that the Broadlink RM device is connected and has transmitted the payload.
 
 Now you can add as many template nodes, each having a specific code, and add any type of input nodes to activate the template and transmit the code.
+
+### {% linkable_title Using broadlink_cli to obtain codes %}
+
+It is also possible to obtain codes using broadlink_cli from https://github.com/mjg59/python-broadlink (you can use my fork which already supports base64 encoding https://github.com/fenio/python-broadlink).
+I tested it only under Linux but it should probably work on other systems as well.
```

Comment:
Documentation should be written in the "third person".

Reply:
rewritten